backend/core/Extensions/DictionaryExtensions.cs

```diff
@@ -217,6 +217,22 @@ public static T GetEnumValue<T>(this IDictionary<string, Microsoft.Extensions.Pr
             return dict.TryGetValue(key, out Microsoft.Extensions.Primitives.StringValues dValue) && Enum.TryParse(typeof(T), dValue, ignoreCase, out object value) ? (T)value : defaultValue;
         }
 
+        /// <summary>
+        /// Get the value from the dictionary for the specified 'key' and return it as an double.
+        /// </summary>
+        /// <param name="dict"></param>
+        /// <param name="key"></param>
+        /// <param name="defaultValue"></param>
+        /// <returns></returns>
+        public static NetTopologySuite.Geometries.Envelope GetEnvelopNullValue(this IDictionary<string, Microsoft.Extensions.Primitives.StringValues> dict, string key, NetTopologySuite.Geometries.Envelope defaultValue = null)
+        {
+            if (!dict.TryGetValue(key, out Microsoft.Extensions.Primitives.StringValues value))
+                return defaultValue;
+
+            var values = value.ToString().Split(',');
+            return NetTopologySuite.Geometries.Envelope.Parse($"Env[{values[0]}:{values[1]},{values[2]}:{values[3]}]");
```

Comment:
Provides a way to deserialize a boundary box (aka envelope).

Reply:
do we want some sanity checking around the number of values so we don't end up with an indexoutofbounds?